classes/local/report.php

```diff
@@ -3735,18 +3735,19 @@ public function get_notes($userid, $gradeitemid, $instn): string {
      * @param object $inst
      * @return string The non-Moodle feedback
      */
-    public function get_turnitin_feedback($userid, $gradeitemid, $inst): stdClass {
+    public function get_turnitin_feedback($userid, $gradeitemid, $inst): string {
```

Comment:
Surely this (and the phpDoc `@return` tag) should be `stdClass|string` as this function returns either:

```
        if ($turnitinfeedback = $currentdb->get_record_sql($sql, $params)) {
            return $turnitinfeedback;
        } else {
            return "";
        }
```